src/storage/exec/MultiTagNode.h

```diff
@@ -48,9 +48,6 @@ class MultiTagNode : public IterateNode<VertexID> {
 
     // add result of each tag node to tagResult
     for (auto* tagNode : tagNodes_) {
```

Comment:
Why remove this?

Reply:
The same Q

Reply:
They have been check on a higher level node, this is redundant. All the check only exists in output node, just to keep unify with others